### ISSUE TITLE
Is there a pre-trained resnet-50?

### ISSUE THREAD

**Issue by Roc-Ng** [qiuqiangkong/audioset_classification#3]
Dear Qiuqiang, is there a resnet-50 model for audio classification? i cannot find it on the Github.

**Comment by qiuqiangkong** [comment]
Many thanks! As the system is built on the AudioSet embedding features, but not on the log mel spectrogram, so we did not apply ResNet50 on it.


Best wishes,


Qiuqiang

________________________________

**Comment by Roc-Ng** [comment]
Thanks for your reply! do u have the resnet-50 model trained on the Audioset or youtube100m? I found u used the resnet-50 as the feature extractor in a weakly supervised setting in your paper...

**Comment by qiuqiangkong** [comment]
Hi, which paper are you referring?


Best wishes,


Qiuqiang

________________________________

**Comment by Roc-Ng** [comment]
MULTI-LEVEL ATTENTION MODEL FOR WEAKLY SUPERVISED AUDIO CLASSIFICATION
Weakly Labelled AudioSet Tagging with Attention Neural Networks
here: The bottleneck features are extracted from the bottleneck layer of a ResNet convolutional neural network~~

**Comment by qiuqiangkong** [comment]
Yes, this is the description of the AudioSet feature. You may find more details in these two paper:

"Audio set: An ontology and human-labeled dataset for audio events"
"CNN Architectures for Large-Scale Audio Classification"


Best wishes,


Qiuqiang

________________________________
